### ISSUE TITLE
Adding new endpoint wss://dstream.binance.com - perpectual coin futures

### ISSUE THREAD

**Issue by oliver-zehentleitner** [oliver-zehentleitner/unicorn-binance-websocket-api#134]
The base endpoint is: wss://dstream.binance.com

https://binance-docs.github.io/apidocs/delivery/en/#change-log

Special handling for userData streams
```
Websocket Request for user data:
    <listenKey>@account request for user's account information
    <listenKey>@balance request for user's account balance
    <listenKey>@balance request for user's position information
```

**Comment by oliver-zehentleitner** [comment]
[Request: User's Position](https://binance-docs.github.io/apidocs/delivery/en/#request-user-39-s-account-balace)
`<listenKey>@position`

**Comment by coinabc123** [comment]
Hi, any updates?

**Comment by xiandong79** [comment]
any updates?

**Comment by oliver-zehentleitner** [comment]
No :/

**Comment by M3tz3l** [comment]
What's the exact problem here @oliver-zehentleitner with the listenkey handling?

**Comment by oliver-zehentleitner** [comment]
Its not a problem, its just not the same as the other private channels.

Now there are 3 types, the handling has to get extended.

**Comment by M3tz3l** [comment]
I tested a little and by just changing the futures websocket baseurl to `dstream` and the enpoint to `dapi` I can receive the [update events](https://binance-docs.github.io/apidocs/delivery/en/#event-balance-and-position-update) just like with USDT margined futures. The new streams are just optional. For me personally, this would be enough for now and should only be small effort. I could open a PR for that if you agree.

**Comment by M3tz3l** [comment]
One can then receive all market data streams and update events via user data stream and extend with support for user data requests later.

**Comment by oliver-zehentleitner** [comment]
@M3tz3l yes that would be great! could you please also provide an example file with tests what is working and what not?

Docs: https://binance-docs.github.io/apidocs/delivery/en/#websocket-market-streams

For example "btcusd_200925@depth" or "btcusd_200925@aggTrade" does a subscription to it work? 

- <symbol>@markPrice OR <symbol>@markPrice@1s
- <pair>@markPrice OR <pair>@markPrice@1s
- <pair>@indexPrice OR <pair>@indexPrice@1s
- <pair>_<contractType>@continuousKline_<interval>
- <pair>@indexPriceKline_<interval>
- ...

**Comment by M3tz3l** [comment]
@oliver-zehentleitner done

**Comment by oliver-zehentleitner** [comment]
could you provide a list of what is not covered from you PR? So we know what we have to finish later.

**Comment by oliver-zehentleitner** [comment]
There is also a testnet: wss://dstream.binancefuture.com'

**Comment by M3tz3l** [comment]
What is missing are the [websocket user data requests](https://binance-docs.github.io/apidocs/delivery/en/#websocket-user-data-request) and coin-m futures testnet support. Would it make sense to create separate issues?

**Comment by oliver-zehentleitner** [comment]
Cool, the public streams are all supported? Generated payloads are all working?

I think one thread/issue is enough.

**Comment by Indiana3714** [comment]
does the stream 'aggTrade' seem to work anyone?

I tried this from the example ".create_stream(['aggTrade'], {'btcusd_perp', 'ethusd_perp', 'bnbusd_perp'})"

but have never gotten back any valid response from the stream buffer
